public/javascripts/data.js

```diff
@@ -148,6 +148,23 @@ var dataNS = odkmaker.namespace.load('odkmaker.data');
         'Vertical Slider': 'vertical',
         'Picker': 'picker'
     };
+    var mediaTypes = {
+         'Image': 'image/*',
+         'Selfie': 'image/*',
+         'Annotate': 'image/*',
+         'Draw': 'image/*',
+         'Signature': 'image/*',
+         'Audio': 'audio/*',
+         'Video': 'video/*',
+         'Selfie Video': 'video/*'
```

Comment:
There's no `Selfie Video` in Collect that I'm aware of. Where'd ya find it?

Reply:
https://github.com/opendatakit/build/issues/200

Reply:
Well, I'll be damned. Checked in the source, and it's definitely there, but it isn't in the docs. I've filed it at https://github.com/opendatakit/docs/issues/982.

In the future, can you link to the issue (if it exists) in the PR. It'd prevent me from embarrassing myself.

Reply:
sorry, it's usually tagged in the commit message but apparently i forgot this time. it _is_ the name of the branch though :)